scripts/workdays.py

```diff
@@ -36,7 +36,7 @@ def main():
         if line.startswith('Total work done this week:'):
             work_in_minutes = parse_time(line.split(':', 1)[1].strip())
             assert work_in_minutes is not None
-            print line,
+            print(line)
```

Comment:
The trailing comma was significant in Python 2!  This introduces a formatting bug.